platform/forms/res/templates/controls/snap-view.html

```diff
@@ -0,0 +1,36 @@
+<!--
+ Open MCT, Copyright (c) 2014-2018, United States Government
+ as represented by the Administrator of the National Aeronautics and Space
+ Administration. All rights reserved.
+
+ Open MCT is licensed under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ License for the specific language governing permissions and limitations
+ under the License.
+
+ Open MCT includes source code licensed under additional open source
+ licenses. See the Open Source Licenses file (LICENSES.md) included with
+ this source code distribution or the Licensing information page available
+ at runtime from the About dialog for additional information.
+-->
+<span ng-controller="SnapshotPreviewController" 
```

Comment:
Why is a new control needed here? How were we doing this previously? Is this just for better code organization?

Reply:
the previous method was using $compile with a new template, which was more complicated than needed, this organizes the code better

Reply:
Can you point me to the code this replaces? I'm having some trouble finding it, and I don't see any code with $compile being removed in this PR?

Reply:
This feature was around in the old implementation of Notebook (master), unfortunately it was not working in the vue implementation of Notebook, so I had to add this to bring back this feature

Reply:
@dtailor90 so do we still need "embedControl"?

Reply:
@larkin no, we don't. I removed those files in the latest commit 3e993a6